src/app/view/components/_common/dot-page-selector/dot-page-selector.component.ts

```diff
@@ -23,19 +29,18 @@ import { take } from 'rxjs/operators';
     ]
 })
 export class DotPageSelectorComponent implements ControlValueAccessor {
-    @Output()
-    selected = new EventEmitter<DotPageAsset>();
-    @Input()
-    style: any;
-    @Input()
-    label: string;
-    @Input()
-    hostIdentifier: string;
-    @Input()
-    floatingLabel = false;
-
-    results: any[];
-    val: DotPageAsset;
+    @Output() selected = new EventEmitter<DotPageAsset>();
+    @Input() style: any;
+    @Input() label: string;
+    @Input() floatingLabel = false;
+
+    results: DotPageSelectorResults = {
+        data: [],
+        query: '',
+        type: ''
+    };
+    val: DotPageSeletorItem;
+    message: string;
 
     constructor(private dotPageSelectorService: DotPageSelectorService) {}
```

Comment:
maybe this service should be into this components's providers, dotPageSelectorService is a state full service so we have to be carefully with its scope